### ISSUE TITLE
edge pixel weight map

### ISSUE THREAD

**Issue by helenxxz** [LeeJunHyun/Image_Segmentation#75]
hey, everyone!
How do you implement the edge pixel weighting in the code?
I only see this:
`
self.criterion = torch.nn.BCELoss()
loss = self.criterion(SR_flat,GT_flat)
` 
Hope to get your answers, thank you!
